### ISSUE TITLE
ONIX display - other values - Audio content types - list 81

### ISSUE THREAD

**Issue by chrisONIX** [w3c/publ-a11y#329]
In the Display techniques for ONIX document - propose to add these values about audio content (access mode audio) from list 81 to the Pre-recorded audio section.
https://ns.editeur.org/onix/en/81

List 81 - code 02 Performance - spoken word (Audio recording of a drama or other spoken word performance)

List 81 - code 13 - other speech content (eg an interview, speech, lecture or commentary / discussion, not a ‘reading’ or ‘performance’))

**Comment by chrisONIX** [comment]
@gautierchomel  shall we close this now?

**Comment by GeorgeKerscher** [comment]
It seems that this issue should be closed.
@chrisONIX 
@gautierchomel 

If this is incorrect, please feel free to reopen or create a new issue.
